helpers/cacheBuild.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+
+const DEFAULT_DIST_DIR = '.next'
+
+// Give advanced/monorepo users the ability to point to their custom build & dist dirs
+const getPath = ({ buildDir, distDir, source }) => {
+  const mainPath = path.join(distDir || DEFAULT_DIST_DIR, source)
```

Comment:
Out of curiosity, when users Next.js directory is not `{buildDir}/.next`, is it because:
  - The directory is not named `.next`?
  - Or the directory is named `.next` but is not located in `buildDir`?

Reply:
the first: directory is not named .next

https://nextjs.org/docs/api-reference/next.config.js/setting-a-custom-build-directory

thats why i wrote in my the PR desc that i technically should get distDir from nextConfig, but i just want to wait for #182 to be merged first (because these 2 PRs conflict)

Reply:
Thanks for the explanation @lindsaylevine 

> i technically should get distDir from nextConfig, but i just want to wait for #182 to be merged first (because these 2 PRs conflict)

After we do this, would there be any use cases left for the `distDir` input?

Reply:
nope, none :)